fitbenchmarking/fitbenchmarking
# Changed rendering of html plotly plots (to reduce size of html files)

## Description of work

Fixes #1183 .


This PR modifies the function used to save the html files for the plotly plots. The reason is to avoid including the whole plotly.js information within the html file as this considerably increases its size. The new function used to save each html file is write_html which allows to point to a single plotly.js file (also added through this PR). This way all the html files for all figures point to the same plotly.js file, rather than repeating the same info.

## Testing Instructions



1. Run fitbenchmarking and check that the plotly plots are shown.
2. Check the size of the html files for the plotly plots in results/support_pages/figures. Each file should be much smaller than 3000KB (e.g.~10-40KB).

## Checklist
